docs/advanced/exceptions.rst

```diff
@@ -79,6 +79,19 @@ This call creates a Python exception class with the name ``PyExp`` in the given
 module and automatically converts any encountered exceptions of type ``CppExp``
 into Python exceptions of type ``PyExp``.
 
+It is possible to specify base class for the exception using the third 
```

Comment:
The trailing space on this line is causing our CI to fail it's "Format" job.

Reply:
It is indeed formatting very well :D fixed

Reply:
A little nitpicky, but now it's passing :-)